Automattic/newspack-ads
# Support for Node v.12, Cleaner Build Script Naming

This PR updates Calypso Build to enable the plugin to be build with current version of Node.js. Also updates the build scripts to match other Newspack Repos:

### Testing 

1) Make sure you are running current node, by  executing `nvm use stable`. `node --version` should show `v12.14.0`
2) Run `npm install`
3) Verify each of these scripts works as expected:
- `npm start`
- `npm run build`
- `npm run release:archive`
4) Verify the description of these scripts in the README is accurate.
